src/registrar/models/domain_request.py

```diff
@@ -467,10 +467,24 @@ class RejectionReasons(models.TextChoices):
     cisa_representative_email = models.EmailField(
         null=True,
         blank=True,
-        verbose_name="CISA regional representative",
+        verbose_name="CISA regional representative email",
         max_length=320,
     )
 
+    cisa_representative_first_name = models.CharField(
```

Comment:
I think it might be worth adding some more tests in the `test_models.py` file, especially in the `TestDomainRequestIncomplete` class (see `test_is_additional_details_complete`) since this is a new additional field that's required within additional details! Sorry for adding more stuff, I just want to be extra careful bc of our request erroring and I know you wanted to be done with this ticket 😢